framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -162,10 +165,8 @@ public void run() {
         LOG.info("[" + testStatus + "] (" + stopwatch + ") " + modeler.queryFilename + " (ConnectionID: " + connection.hashCode() + ")");
       }
 
-      if((++countTestsCompleted %100==0 && countTestsCompleted <= totalCases) || (countTestsCompleted == totalCases)){
-        LOG.info(LINE_BREAK);
-        LOG.info("Execution completed for " + countTestsCompleted + " (out of " + totalCases + ") tests");
-        LOG.info(LINE_BREAK);
+      if((++countTestsCompleted %100==0 && countTestsCompleted <= (totalCases*TestDriver.cmdParam.iterations*TestDriver.cmdParam.clones))){
```

Comment:
Isn't countTestsCompleted per iteration? If not it should be. That way we can keep the check as is.

Reply:
it is not per iteration ,we have to change the logic for it ,this change is due to the fact that it is not an iteration specific thing thus ,the "Execution completed for  ..." is not printed for subsequent iterations